deploy/crds/operators_v1_catalogsourceconfig_crd.yaml

```diff
@@ -30,6 +30,7 @@ spec:
     JSONPath: .metadata.creationTimestamp
   validation:
     openAPIV3Schema:
+      description: CatalogSourceConfig configures a CatalogSource with information about available operators.
```

Comment:
Please use the description from the README.

Reply:
Ah will do! Thanks!